test-file-trees/bad-structure/cli/invalid-option-accepts/invalid-option-accepts.spec.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+	description: 'Order a pizza, to-go',
+	data: {
+		description: 'What type of pizza to order',
+	},
+	options: {
+		test: {
+			description: 'Just used for testing',
+			type: 'fake',
```

Comment:
Why is this set to `fake`? The directory name you chose implies that the problem is an invalid `accepts` value, but here you're testing an invalid `type` value...

Reply:
I did this as part of my testing.  A bit unrelated, but, while TypeScript is good at making sure data matches expected values, since this is not only to be used by TypeScript clients, it goes to show that invalid data can be provided and sneak its way into the application without being guarded against.  Seems some validation for this value being one of the expected values, and erroring if not, is a good thing to add.